rpm/proxysql.spec

```diff
@@ -73,14 +75,17 @@ install -m 0640 %SOURCE3 %{buildroot}/%{_sysconfdir}
 
 install -d %{buildroot}/var/lib/proxysql
 install -d %{buildroot}/var/run/proxysql
+install -d %{buildroot}/var/run/pxc_scheduler_handler
 install -m 0775 %SOURCE1 %{buildroot}/%{_bindir}/proxysql-admin
 install -m 0775 %SOURCE8 %{buildroot}/%{_bindir}/proxysql-admin-common
+install -m 0775 %SOURCE11 %{buildroot}/%{_bindir}/proxysql-common
```

Comment:
One question: Why the file permission is 0775, why not 0755. All standard packages have 0755.

Reply:
It looks like a misprint in initial commits.
I will fix permissions. 
<img width="710" alt="Screenshot 2022-06-09 at 21 40 35" src="https://user-images.githubusercontent.com/25184101/172920827-a9a60a72-5e46-4ee2-9b32-04e129479571.png">